compute_endpoint/globus_compute_endpoint/executors/high_throughput/worker.py

```diff
@@ -180,6 +181,10 @@ def call_user_function(self, message: bytes) -> str:
             task_data = task.task_buffer.decode("utf-8")  # type: ignore[attr-defined]
 
         f, args, kwargs = self.serializer.unpack_and_deserialize(task_data)
+        GC_TASK_TIMEOUT = int(os.environ.get("GC_TASK_TIMEOUT", 0))
+        if GC_TASK_TIMEOUT > 0:
```

Comment:
Though second-granular timing is likely "just fine" for the user use-cases, I observe that Timer expects a `float`.  For out _tests_, if we accepted a float, we could reduce the idle second waiting for the test.  Consider:
```python
GC_TASK_TIMEOUT = max(0.0, float(os.environ.get("GC_TASK_TIMEOUT", 0.0)))
```
Then the test could use much smaller values.

Reply:
I've updated it to accept floats, and the tests run faster.